conda/meta.yaml

```diff
@@ -33,8 +33,8 @@ test:
   commands:
     - zppy --help
     - pip check
-
 about:
+
   home: https://github.com/E3SM-Project/zppy
```

Comment:
I don't think you want this white-space change.  It's at least not conventional.

Reply:
Or maybe I did? lol does look weird.